src/transaction/log_page_buffer.c

```diff
@@ -1525,9 +1055,8 @@ logpb_get_page_id (LOG_PAGE * log_pgptr)
 {
```

Comment:
It's possible but impractical because usages are only debugging support functions. 
Please forget it.